packages/next/package.json

```diff
@@ -277,6 +279,7 @@
     "terser": "5.14.1",
     "text-table": "0.2.0",
     "timers-browserify": "2.0.12",
+    "ts-morph": "17.0.1",
```

Comment:
This can't be added because it's way too big: https://packagephobia.com/result?p=ts-morph